models/vehicle.go

```diff
@@ -3,21 +3,25 @@ package models
+	BodyType           string    `firestore:"bodyType,omitempty"           json:"bodyType,omitempty"           bigquery:"-"` // BODY TYPE (sedan, hatchback, suv, coupe, convertible, wagon, minivan, pickup truck, van, crossover)
+	Setup              string    `firestore:"setup,omitempty"              json:"setup,omitempty"              bigquery:"-"` // ALLESTIMENTO (cilindrata, kw/cv)
+	OwnerType          string    `firestore:"ownerType,omitempty"          json:"ownerType,omitempty"          bigquery:"-"` // TIPOLOGIA PROPRIETARIO (uomo/donna/società-p.iva)
+	HasTowHook         bool      `firestore:"hasTowHook,omitempty"         json:"hasTowHook,omitempty"         bigquery:"-"` // PRESENZA GANCIO TRAINO
+	KmPerYear          int64     `firestore:"kmPerYear,omitempty"          json:"kmPerYear,omitempty"          bigquery:"-"` // KM ANNUI
+	OvernightShelter   string    `firestore:"overnightShelter,omitempty"   json:"overnightShelter,omitempty"   bigquery:"-"` // RICOVERO NOTTURNO (box privato/garage pubblico/area recintata privata/in strada)
+	MainUse            string    `firestore:"mainUse,omitempty"            json:"mainUse,omitempty"            bigquery:"-"` // UTILIZZO PRINCIPALE (tempo libero/tempo libero e casa-lavoro/lavoro)
 	AlarmTypeInstalled string    `firestore:"alarmTypeInstalled,omitempty" json:"alarmTypeInstalled,omitempty" bigquery:"-"` // ANTIFURTO INSTALLATO (nessuno/meccanico/elettronico/satellitare)
-	OwnershipStatus    string    `firestore:"ownershipStatus,omitempty" json:"ownershipStatus,omitempty" bigquery:"-"`       // GIA’ PROPRIETARIO (si/in attesa/la comprerò in futuro)
-	NumberOfOwners     int64     `firestore:"numberOfOwners,omitempty" json:"numberOfOwners,omitempty" bigquery:"-"`         // NUMERO DI PROPRIETARI
-	PriceValue         int64     `firestore:"priceValue,omitempty" json:"priceValue,omitempty" bigquery:"-"`                 // VALORE VEICOLO
+	OwnershipStatus    string    `firestore:"ownershipStatus,omitempty"    json:"ownershipStatus,omitempty"    bigquery:"-"` // GIA’ PROPRIETARIO (si/in attesa/la comprerò in futuro)
+	NumberOfOwners     int64     `firestore:"numberOfOwners,omitempty"     json:"numberOfOwners,omitempty"     bigquery:"-"` // NUMERO DI PROPRIETARI
+	PriceValue         int64     `firestore:"priceValue,omitempty"         json:"priceValue,omitempty"         bigquery:"-"` // VALORE VEICOLO
+	VehicleType        string    `firestore:"vehicleType,omitempty"        json:"vehicleType,omitempty"        bigquery:"-"` // TIPO VEICOLO
+	Weight             float64   `firestore:"weight,omitempty"             json:"weight,omitempty"             bigquery:"-"` // PESO VEICOLO IN QUINTALI
+	PowerSupply        string    `firestore:"powerSupply,omitempty"        json:"powerSupply,omitempty"        bigquery:"-"` // ALIMENTAZIONE
+	HasSatellite       bool      `firestore:"hasSatellite,omitempty"       json:"hasSatellite,omitempty"       bigquery:"-"`
 }
```

Comment:
Move down in a comment all fields that are not currently used e.g:

- Vin
- AlarmTypeInstalled
- Year
- BodyType
- Setup
- OwnerType
- HasTowHook
- KmPerYear
- OvernightShelter
- MainUse